src/Dfc.CourseDirectory.FindACourseApi/Features/TLevelDetail/TLevelLocationViewModel.cs

```diff
@@ -1,7 +1,10 @@
-﻿namespace Dfc.CourseDirectory.FindACourseApi.Features.TLevelDetail
+﻿using System;
+
+namespace Dfc.CourseDirectory.FindACourseApi.Features.TLevelDetail
 {
     public class TLevelLocationViewModel
     {
+        public Guid TLevelLocationId { get; set; }
```

Comment:
I can't see any updates to the `SELECT` statements in any SqlQueryHandlers to populate this value?

Reply:
As far as I can see this property is already populated by the query handlers.

Reply:
Ignore me, I misunderstood that class.